src/main/java/entity/Stock.java

```diff
@@ -29,16 +29,16 @@ public String getIndustry() {
         return industry;
     }
 
-    public double getPrice() {
-        return price;
+    public double getMarketPrice() {
+        return marketPrice;
     }
 
     /**
-     * Update the current market price of the stock
+     * Update the current market executionPrice of the stock
```

Comment:
I don't think here we should use `executionPrice`. Should it be `marketPrice`? What is this method used for? Maybe fixed the @javadoc

Reply:
This typo is made when doing refactoring globally, you can fix that. These may also exist in other files.